### PR TITLE
Image display on map

### PR DESCRIPTION
**Pull request by KShih**
Enable the map to show the pic of the apartment if there is one available on its URL.

### CODE REVIEW

**Comment by ab7289** [comment]
I am getting connection errors when I try to view the map after retrieving CraigsListLocations.

image

File: map/views.py [hunk 1]
```diff
@@ -7,23 +10,63 @@ def mapview(request):
...
-        title = data.name
-        url = '<a href="' + data.url + '" target="_blank">' + data.url + "</a>"
-        description = data.price + "\n" + str(data.date_time) + "\n" + url
+        if not CityStreetSpot.objects.filter(c_id=data.c_id).exists():
+            title = data.name
+            url = '<a href="' + data.url + '" target="_blank">' + data.url + "</a>"
+            description = data.price + "\n" + str(data.date_time) + "\n" + url
+
+            if data.has_image:
+                image_url = get_img_url(data.url)
```
Comment (ab7289) [inline map/views.py]: Would it make more sense to load the image when we retrieve the CraigslistLocatin instead of here when we're trying to load the map?

Comment (KShih) [reply map/views.py]: Yes, it makes more sense. Will do that next sprint! Thanks!

**Comment by ab7289** [comment]
Since this branch has become out of date from develop and it is failing the build I think we will have to wait until next sprint for this, especially since Jeff doesn't have a computer.

**Pull request closed without merging**